tests/data/LF_lat_lon_UseCase/reference_ricewateruseOFF/chanqWin.tss

```diff
@@ -0,0 +1,29 @@
+timeseries valuescale.scalar settingsfile: /BGFS/DISASTER/grimast/MERGE_allocate_unittests/MASTERv3.1_plusMerge/lisflood-code/tests/data/LF_lat_lon_UseCase/run_lat_lon.xml date: Fri Sep 10 15:53:42 2021
```

Comment:
The folder tests/data/LF_lat_lon_UseCase/**reference_ricewateruseOFF**/ can be deleted before the push.